homeassistant/components/device_tracker/ping.py

```diff
@@ -0,0 +1,116 @@
+"""
+Support for detect network host with ping.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.ping/
+
+device_tracker:
+  - platform: ping
+    count: 2
+    hosts:
+      - 192.168.2.10
+      - 192.168.2.25
+"""
+
+import logging
+import subprocess
+import re
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import (
+    PLATFORM_SCHEMA, DEFAULT_SCAN_INTERVAL)
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant import util
+from homeassistant import const
+import homeassistant.helpers.config_validation as cv
+
+DEPENDENCIES = []
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_PING_COUNT = 'count'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(const.CONF_HOSTS): cv.ensure_list,
+    vol.Optional(CONF_PING_COUNT, default=2): cv.positive_int,
+})
+
+
+class Host:
+    """Host with ping and mac detection"""
+
+    def __init__(self, ip_address, hass, config):
+        self.hass = hass
+        self.ip_address = ip_address
+        self.mac = ''
+        self.dev_id = ''
+        self._count = config[CONF_PING_COUNT]
+        self._ping_cmd = ['ping', '-q', '-W1',
+                          '-c{}'.format(self._count), self.ip_address]
+        self._arp_cmd = ['arp', '-n', self.ip_address]
```

Comment:
If you are already going to hardcode the IP addresses, why not just get the mac addresses too? Or, for what it is worth, you can also pass a `dev_id` instead of a mac to the `see` function. 

That way your config could look like this:

```yaml
device_tracker:
  - platform: ping
    count: 3 # number of packet used for each device (avoid false detection)
    hosts:
      my_phone: 192.168.2.10
      my_pc: 192.168.2.25
```

my_phone / my_pc in this case would be used as dev_id